packages/apps-config/src/endpoints/production.ts

```diff
@@ -214,7 +214,8 @@ export const prodChains: EndpointOption[] = [
     text: 'Polkadex',
     providers: {
       'Polkadex Team': 'wss://mainnet.polkadex.trade',
-      OnFinality: 'wss://polkadex.api.onfinality.io/public-ws'
+      OnFinality: 'wss://polkadex.api.onfinality.io/public-ws',
+      Pinknode: 'wss://public-rpc.pinknode.io/polkadex',
```

Comment:
Extra trailing , making the he checks unhappy.

Reply:
Removed it 👍